validphys2/src/validphys/config.py

```diff
@@ -213,6 +212,10 @@ def parse_use_cuts(self, use_cuts: (bool, str)):
 
         return res
 
+    def parse_with_shift(self, with_shift: bool):
+        """Shift to theory due to correlated systematics in data/theory plots"""
+        return with_shift
+               
```

Comment:
```suggestion
```

I believe this is not needed since it is just a `True/False` flag? Or am I missing something?

Reply:
Done